app/src/main/res/layout/reading_text_size_fragment.xml

```diff
@@ -9,42 +9,6 @@
       type="org.oppia.android.app.options.ReadingTextSizeSelectionViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
```

Comment:
I am not sure whether we have to remove both Constraint and Frame Layout from here, though we know only keeping a RecylerView makes a lot of sense. @rt4914 what are your reviews over this?